### PR TITLE
[homework#3] HTML-CSS by Mariana Dzividzinska

### PR DESCRIPTION
**Pull request by DzividzinskaM**
First task result:
image

Second task result: 
image

### CODE REVIEW

File: homeworks/mariana-dzividzinska/03-html-css/task1/index.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                    </p>
+                    <p class="about-us__description">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Non tempora laudantium deserunt
+                        nesciunt magnam natus fugit placeat, ea quis quos qui libero sequi sed id tempore inventore,
+                        provident impedit nemo. Lorem, ipsum dolor sit amet consectetur adipisicing elit. Non tempora laudantium deserunt
+                        nescLorem, ipsum dolor sit amet consectetur adipisicing elit.
+                    </p>
+                </article>
+    
+                <article class="portfolio">
+                    <h1 class="title">Portfolio</h1>
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task1/index.html:53]: Only one `<h1>` tag containing the main page title should be on the page. Please try this tool to check your page structure https://gsnedders.html5.org/outliner/

File: homeworks/mariana-dzividzinska/03-html-css/task1/index.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                            <h2 class="project__title">Project name</h2>
+                            <div class="project__description">Lorem ipsum dolor sit amet consectetur adipisicing elit.</div>
+                        </section>
+                        <section class="portfolio__project">
+                            <img src="https://fakeimg.pl/190x100/" alt="" class="project__img">
+                            <h2 class="project__title">Project name</h2>
+                            <div class="project__description">Lorem ipsum dolor sit amet consectetur adipisicing elit.</div>
+                        </section>
+                    </div>
+
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task1/index.html:86]: redundant empty line

File: homeworks/mariana-dzividzinska/03-html-css/task1/index.html [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                        <section class="portfolio__project">
+                            <img src="https://fakeimg.pl/190x100/" alt="" class="project__img">
+                            <h2 class="project__title">Project name</h2>
+                            <div class="project__description">Lorem ipsum dolor sit amet consectetur adipisicing elit.</div>
+                        </section>
+                    </div>
+                    
+                </article>
+            </div>
+
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task1/index.html:89]: redundant empty line

File: homeworks/mariana-dzividzinska/03-html-css/task1/index.html [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        </main>
+       
+        <div class="delimiter"></div>
+    
+        <footer class="footer">
+            <p>(c) Development laba 2021</p>
+        </footer>
+    </div>
+</body>
+</html>
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task1/index.html:99]: incorrect indent: `<html>` wraps `<body>`

File: homeworks/mariana-dzividzinska/03-html-css/task1/style.css [hunk 5]
```diff
@@ -0,0 +1,107 @@
+/* html, body {
+    box-sizing: border-box;
+    font-size: 10px;
+}
+
+*, *::after, *::before {
+    box-sizing: inherit;
+} */
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task1/style.css:8]: remove unused code

File: homeworks/mariana-dzividzinska/03-html-css/task1/style.css [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+ul li {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+.container {
+    width: 810px;
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task1/style.css:27]: Incorrect width

File: homeworks/mariana-dzividzinska/03-html-css/task1/style.css [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+    padding: 5px 10px;
+    background: #f1f1f1;
+    margin: 10px 0;
+}
+
+.content {
+    float: right;
+    width: 600px;
+}
+
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task1/style.css:83]: redundant empty line

File: homeworks/mariana-dzividzinska/03-html-css/task1/index.html (hunk 1 above)
Comment (danymill) [reply homeworks/mariana-dzividzinska/03-html-css/task1/index.html:53]: Also, the HTML validator shows you appropriate warnings 
image

File: homeworks/mariana-dzividzinska/03-html-css/task2/cv.html [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+                    </li>
+                </ul>
+            </article>
+        </aside>
+        
+        <main class="main-info">
+            <header class="header">
+                <h1 class="header__name">Mariana Dzividzinska</h1>
+            </header>
+            <article class="education">
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task2/cv.html:36]: Please [read this](https://habr.com/ru/post/546994/) article, maybe you will change your mind to use `<article>` here, maybe not :)

File: homeworks/mariana-dzividzinska/03-html-css/task2/style.css [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+
+body {
+    color: #373741;
+    margin: 0;
+}
+
+a {
+    color: #373741;
+}
+
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task2/style.css:14]: redundant empty line

File: homeworks/mariana-dzividzinska/03-html-css/task2/style.css [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+    margin-top: 20px;
+}
+
+.cv__photo img {
+    width: 150px;
+    height: 150px;
+    border-radius: 50%;
+    border: 2px solid #5c2020fd;
+}
+
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task2/style.css:56]: redundant empty line

File: homeworks/mariana-dzividzinska/03-html-css/task2/style.css [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.header__name{
+    font-size: 3.2rem;
+    color: #e1d8cf;
+}
+
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task2/style.css:89]: redundant empty line

File: homeworks/mariana-dzividzinska/03-html-css/task2/style.css [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+
+
+
+
+
+
+
+
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task2/style.css:135]: too many empty lines

File: homeworks/mariana-dzividzinska/03-html-css/task2/style.css [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+    margin: 0 auto;
+    display: flex;
+    height: 100vh;
+}
+
+.title {
+    width: 200px;
+    height: 20px;
+    font-size: 1.8rem;
+    background: #5c2020fd;
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task2/style.css:27]: incorrect color ([read this](https://www.w3schools.com/css/css_colors_hex.asp))

File: homeworks/mariana-dzividzinska/03-html-css/task2/style.css [hunk 14]
```diff
@@ -0,0 +1,135 @@
...
+
+.cv__photo {
+    margin-top: 20px;
+}
+
+.cv__photo img {
+    width: 150px;
+    height: 150px;
+    border-radius: 50%;
+    border: 2px solid #5c2020fd;
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task2/style.css:54]: incorrect color

File: homeworks/mariana-dzividzinska/03-html-css/task2/style.css [hunk 15]
```diff
@@ -0,0 +1,135 @@
...
+}
+
+.main-info article {
+    margin-left: 20px;
+}
+
+.header {
+    margin-top: 20px;
+    height: 150px;
+    background-color: #5c2020fd;
```
Comment (danymill) [inline homeworks/mariana-dzividzinska/03-html-css/task2/style.css:79]: incorrect color

File: homeworks/mariana-dzividzinska/03-html-css/task1/index.html (hunk 4 above)
Comment (danymill) [reply homeworks/mariana-dzividzinska/03-html-css/task1/index.html:99]: But if you used some tool, or _emmet_ to generate HTML template, please skip this place, but check other indents

**Review by danymill — CHANGES_REQUESTED** [review]

**Comment by danymill** [comment]
Please note, that you didn't complete the last task (selected on screen) 
image

**Comment by DzividzinskaM** [comment]
I've already fixed issues

**Pull request merged**